package.json

```diff
@@ -32,14 +32,14 @@
     "@ionic-native/core": "^5.36.0",
     "@ionic-native/splash-screen": "^5.0.0",
     "@ionic-native/status-bar": "^5.0.0",
-    "@ionic/angular": "^5.9.1",
+    "@ionic/angular": "^6.1.8",
     "@ionic/core": "^5.9.1",
```

Comment:
Check if the core needs to be updated as well.

There is also a dev dependency called `@ionic/angular-toolkit` that might need to be updated;